test/e2e/config/ironic-vbmc.yaml

```diff
@@ -0,0 +1,73 @@
+images:
+# Use locally built e2e images
+- name: quay.io/metal3-io/baremetal-operator:e2e
+  loadBehavior: tryLoad
+- name: quay.io/metal3-io/ironic:e2e
+  loadBehavior: tryLoad
+# Save some time and network by using cached images if available
+- name: quay.io/metal3-io/baremetal-operator:release-0.4
+  loadBehavior: tryLoad
+- name: quay.io/jetstack/cert-manager-cainjector:v1.13.1
+  loadBehavior: tryLoad
+- name: quay.io/jetstack/cert-manager-webhook:v1.13.1
+  loadBehavior: tryLoad
+- name: quay.io/jetstack/cert-manager-controller:v1.13.1
+  loadBehavior: tryLoad
+
+# Test credentials. The tests will create a BMH with these.
+# During development, select the preferred BMC interface when running tests from the command line:
+bmcs:
+- user: admin
+  password: password
+  address: "ipmi://192.168.222.1:16230"
+  bootMacAddress: "00:60:2f:31:81:01"
+  hostName: "bmo-e2e-0"
+  sshAddress: "192.168.222.122:22"
+- user: admin
+  password: password
+  address: "ipmi://192.168.222.1:16231"
+  bootMacAddress: "00:60:2f:31:81:02"
+  hostName: "bmo-e2e-1"
+  sshAddress: "192.168.222.123:22"
+
+# These variables can be overridden with environment variables.
+variables:
+  # These settings are for the "default" cluster, which is used for all tests except upgrade ones
+  DEPLOY_IRONIC: "true"
+  DEPLOY_BMO: "true"
+  DEPLOY_CERT_MANAGER: "true"
+  BMO_KUSTOMIZATION: "../../config/overlays/e2e"
+  IRONIC_KUSTOMIZATION: "../../ironic-deployment/overlays/e2e"
+
+  # These settings are for the separate cluster that are used for upgrade tests
+  UPGRADE_DEPLOY_IRONIC: "true"
+  UPGRADE_DEPLOY_BMO: "true"
+  UPGRADE_DEPLOY_CERT_MANAGER: "true"
+  UPGRADE_BMO_KUSTOMIZATION_FROM: "../../config/overlays/e2e-release-0.4"
```

Comment:
We do upgrade from 0.4? Why not from 0.5?

Reply:
These two new files are direct replacement of the `ironic.yaml`, so the content was just copied over. Using another upgrade from version is another story. Please check https://github.com/metal3-io/baremetal-operator/pull/1523